src/clojure/nrepl/transport.clj

```diff
@@ -73,8 +81,16 @@
        (map (fn [[k v]] [k (<bytes v)]))
        (into {})))
 
+(set! *warn-on-reflection* false)
+
+(defn- connected? [s] (.isConnected s))
+(defn- flush-output [s] (.flush s))
+(defn- close-stream [s] (.close s))
```

Comment:
Why the reflection?

Reply:
Good question, and it turns out to be unnecessary.